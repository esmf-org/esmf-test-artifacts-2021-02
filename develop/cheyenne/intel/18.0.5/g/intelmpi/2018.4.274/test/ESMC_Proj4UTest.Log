20210101 155916.886 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210101 155916.886 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210101 155916.886 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210101 155916.886 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210101 155916.886 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210101 155916.886 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210101 155916.886 INFO             PET0 Running with ESMF Version   : ESMF_8_1_0_beta_snapshot_40-4-ga072c96c19
20210101 155916.886 INFO             PET0 ESMF library build date/time: "Jan  1 2021" "15:08:17"
20210101 155916.886 INFO             PET0 ESMF library build location : /glade/scratch/himanshu/scripts_dir/daily/intel_cheyenne/esmf
20210101 155916.886 INFO             PET0 ESMF_COMM                   : intelmpi
20210101 155916.887 INFO             PET0 ESMF_NETCDF                 : disabled
20210101 155916.887 INFO             PET0 ESMF_PNETCDF                : disabled
20210101 155916.887 INFO             PET0 ESMF_PIO                    : enabled
20210101 155916.887 INFO             PET0 ESMF_YAMLCPP                : enabled
20210101 155916.887 INFO             PET0 ESMF_MOAB                   : enabled
20210101 155916.887 INFO             PET0 Beginning Test, file ESMC_Proj4UTest.C, line 49
20210101 155916.887 INFO             PET0 NUMBER_OF_PROCESSORS 1
20210101 155916.887 INFO             PET0 PASS PROJ.4 rotated pole transform test, ESMC_Proj4UTest.C, line 119
20210101 155916.887 INFO             PET0 Ending Test, file ESMC_Proj4UTest.C, line 123
20210101 155916.887 INFO             PET0 Finalizing ESMF
Fri Jan  1 18:07:08 MST 2021
