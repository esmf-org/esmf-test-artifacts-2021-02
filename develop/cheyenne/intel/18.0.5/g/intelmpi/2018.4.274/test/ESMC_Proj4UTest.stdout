Beginning Test, file ESMC_Proj4UTest.C, line 49
NUMBER_OF_PROCESSORS 1
PASS PROJ.4 rotated pole transform test, ESMC_Proj4UTest.C, line 119
Ending Test, file ESMC_Proj4UTest.C, line 123
 PET 0 Test Elapsed Time  0.827000 msec.
Thu Jan  7 20:50:00 MST 2021
